spring-cloud-netflix-core/pom.xml

```diff
@@ -202,6 +202,18 @@
 			<artifactId>jackson-dataformat-smile</artifactId>
 			<optional>true</optional>
 		</dependency>
+		<dependency>
```

Comment:
Can we do this without adding these dependencies?

Reply:
A Protobuf class need to be created in the test, so I think these two test dependencies are required, otherwise it can not be compiled.

Reply:
@ryanjbaxter 

The Feign’s functionally is based on Spring, which uses same dependencies and tests in Spring Web. I think we should add these into Feign as well.

refer:

https://github.com/spring-projects/spring-framework/blob/master/spring-web/spring-web.gradle#L63-L64
https://github.com/spring-projects/spring-framework/blob/master/spring-web/src/test/java/org/springframework/http/converter/protobuf/ProtobufHttpMessageConverterTests.java